tests/integration/main.rs

```diff
@@ -0,0 +1,48 @@
+mod artichoke;
+
+use std::env;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+#[allow(dead_code)]
+#[derive(Debug)]
+struct CommandOutput<'a> {
+    call_args: Vec<&'a str>,
+    status: i32,
+    stdout: String,
+    stderr: String,
```

Comment:
Let's make this change after adding a `use bstr::BString` above.

```suggestion
    stdout: BString,
    stderr: BString,
```